src/include/partitioning/custom_part_check_generated.rb

```diff
@@ -294,6 +299,20 @@ def check_created_partition_table(targetMap, installation)
         ok = false if !Popup.YesNo(message)
       end
 
+      # A PReP/CHRP partition is not supposed to be mounted. So if we find any
+      # other /boot partition, we should warn the user.
+      if boot_found && Partitions.PrepBoot && !boot_mount_point.empty? && !Arch.board_iseries && installation || show_all_popups
```

Comment:
@imobach Although this probably works, it's quite a tricky condition ;) BTW why do we have !Arch.board_iseries there? It's partly documented, and this is missing. Shouldn't we rather use Mode.installation?